### PR TITLE
Update apps-id.md

### PR DESCRIPTION
**Pull request by traci707**
# [DEVDOCS-2287](https://jira.bigcommerce.com/browse/DEVDOCS-2287)

## What changed?

- Changed devTools to Developer Portal
- Fixed the repeating sections

### CODE REVIEW

File: docs/api-docs/apps/tutorials/apps-id.md [hunk 1]
```diff
@@ -4,15 +4,9 @@
 
 ### On this page
 - [Find in control panel](#find-in-control-panel)
-- [Find in DevTools](#find-in-devtools)
+- [Find in developer portal](#find-in-devtools)
```
Comment (ektravel) [inline docs/api-docs/apps/tutorials/apps-id.md:7]:
```suggestion
- [Find in Developer Portal](#find-in-devtools)

```

File: docs/api-docs/apps/tutorials/apps-id.md [hunk 2]
```diff
@@ -34,7 +28,7 @@ The app ID is in the URL after `/manage/marketplace/apps/`:
 
 ![Draft App ID](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/apps-id-02.png "Draft App ID")
 
-## Find in DevTools
+## Find in developer portal
```
Comment (ektravel) [inline docs/api-docs/apps/tutorials/apps-id.md:31]:
```suggestion
## Find in Developer Portal

```

**Review by ektravel — COMMENTED** [review]
Finished my review.

File: docs/api-docs/apps/tutorials/apps-id.md [hunk 3]
```diff
@@ -4,15 +4,10 @@
 
 ### On this page
 - [Find in control panel](#find-in-control-panel)
-- [Find in DevTools](#find-in-devtools)
-- [Usage](#usage)
-- [Resources](#resources)
+- [Find in Developer Portal](#find-in-devtools)
```
Comment (aglensmith) [inline docs/api-docs/apps/tutorials/apps-id.md:8]: Remove empty line on line 8.

**Review by aglensmith — CHANGES_REQUESTED** [review]
Remove empty line (line 8).

File: docs/api-docs/apps/tutorials/apps-id.md [hunk 4]
```diff
@@ -34,7 +29,8 @@ The app ID is in the URL after `/manage/marketplace/apps/`:
 
 ![Draft App ID](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/apps-id-02.png "Draft App ID")
 
-## Find in DevTools
+## Find in Developer Portal
+
```
Comment (aglensmith) [inline docs/api-docs/apps/tutorials/apps-id.md:33]: Remove this extra line.

**Review by aglensmith — CHANGES_REQUESTED** [review]

**Review by uwikaiddi — APPROVED** [review]

**Pull request merged**